frontend/components/contextSnippets/ProgrammaticAccessMenu.tsx

```diff
@@ -52,7 +52,7 @@ export const ProgrammaticAccessMenu = () => {
           <Menu.Button as={Fragment}>
             <Button variant="secondary" title="One-click programmatic access">
               <FaTerminal className="text-emerald-500" />
-              <span className="font-mono">Access</span>
+              <span className="font-mono">One-click Access</span>
```

Comment:
```suggestion
             One-click Access
```

I'd suggest removing the monospace font for the longer button label